client/state/selectors/get-customize-or-edit-front-page-url.js

```diff
@@ -0,0 +1,30 @@
+/** @format */
+
+/**
+ * Internal dependencies
+ */
+import { getThemeCustomizeUrl, isThemeActive } from 'state/themes/selectors';
+import isSiteUsingFullSiteEditing from 'state/selectors/is-site-using-full-site-editing';
+import getFrontPageEditorUrlUrl from 'state/selectors/get-front-page-editor-url';
+
+/**
+ * Returns the URL for opening customizing the given site in either the block editor with
+ * Full Site Editing, or the Customizer for unsupported sites. Can be used wherever
+ * @see getThemeCustomizeUrl is used as a drop-in replacement.
+ *
+ * Ensure that your view makes use of the `QueryBlogStickers` component to function properly.
+ *
+ * @param  {Object}   state   Global state tree
+ * @param  {String}   themeId Theme ID
+ * @param  {Number}   siteId  Site ID to open the customizer or block editor for
+ * @return {String}           Customizer or Block Editor URL
+ */
+export default function getCustomizeOrEditFrontPageUrl( state, themeId, siteId ) {
```

Comment:
Mind also adding some basic tests for the new selectors? There should be many examples in the folder (nested in test)